packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -88,19 +90,34 @@ function NavigationMenuSelector(
 				<>
 					{ showNavigationMenus && hasNavigationMenus && (
 						<MenuGroup label={ __( 'Menus' ) }>
-							<MenuItemsChoice
```

Comment:
Removing this will take out the tick when the menu is selected, causing some confusion.